kotlin-mui/src/jsMain/generated/mui/base/useList.types.kt

```diff
@@ -174,3 +174,15 @@ external interface ListItemState {
      */
     var selected: Boolean
 }
+
+external interface UseListReturnValue {
+    var contextValue: dynamic
+
+    var dispatch: (action: Any /* CustomAction | ListAction<ItemValue> */) -> Unit
+
+    var getRootProps: react.Props /* <TOther extends EventHandlers = {}>(otherHandlers?: TOther) => UseListRootSlotProps<TOther> */
```

Comment:
Looks like function

Reply:
It should be a separate change for all functions like this one. Next time